lib/editor/components/map/TextPath.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import { Path } from 'react-leaflet'
+import L from 'leaflet'
+import 'leaflet-textpath'
+
+// Force polyline to be non-interactive and invisible.
+const OPTIONS = {
+  interactive: false,
+  weight: 0
+}
+
+type Props = {
+  attributes: Object,
+  below: boolean,
+  center: boolean,
+  leaflet: any,
+  offset?: number,
+  orientation?: number,
+  positions: Array<[number, number]>,
+  repeat: boolean,
+  text: string
+}
+
+// props.leaflet is not used but destructured out so it's not passed to L.Polyline
+function getOptions ({
```

Comment:
Can you use [lodash.omit](https://lodash.com/docs/4.17.11#omit) for this instead?